docs/kb/semgrep-ci/trigger-diff-scans-env-var.md

```diff
@@ -118,7 +122,7 @@ jobs:
 </TabItem>
 <TabItem value='gitlab'>
 
-Obtain the value of `$CI_MERGE_REQUEST_IID`, the unique project-level IID (internal ID) of the merge request, in the `rules` section of the pipeline definition. The results of the evaluation determine the attributes of the job. If `$CI_MERGE_REQUEST_IID` exists, Semgrep runs a diff-aware scan:
+Set up your `.gitlab-ci.yml` conditions (usually `rules`) to run a scan if `$CI_MERGE_REQUEST_IID` is defined. Semgrep automatically runs a diff-aware scan if the variable is present, as it is in merge request pipelines:
```

Comment:
```suggestion
Set up your `.gitlab-ci.yml` conditions (usually `rules`) to run a scan if `$CI_MERGE_REQUEST_IID` is defined. Semgrep automatically runs a diff-aware scan if the variable is present, as it does in merge request pipelines:
```

Not sure if I'm understanding this correctly?

Reply:
Yeah, I think there's a semantic nuance here. The variable **is** present in merge request pipelines, and therefore Semgrep does run a diff-aware scan there.